toDo_API_Spec.md

```diff
@@ -0,0 +1,81 @@
+# ToDo API
+---
+This is a proposal for my API Spec 
+## End Points
+---
+
+### Create ToDo
+`POST .com/ToDo`
```

Comment:
the `.com` was only there to help us bridge the gap between website URL and API URL in an actual spec please only include the URL for the endpoint (start with `/`)